Tasks/PowerShellV2/task.json

```diff
@@ -16,8 +16,8 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 2,
-        "Minor": 163,
+        "Major": 3,
```

Comment:
Why are you bumping the major version of this task? These seems like a backwards compatible change.

Reply:
I thought initially but now I will change only minor version and update the same. Is that fine?

Reply:
I revert back the major and minor version but I changed only the patch. Let me know is this fine or need to change minor version as well.

Reply:
That's fine. Can you rebase your change as well?

Reply:
Done. Please check the recent commit and approve.